contracts/StandardBounty.sol

```diff
@@ -0,0 +1,319 @@
+pragma solidity ^0.4.18;
+import "./inherited/StandardToken.sol";
+
+/// @title StandardBounties
+/// @dev Used to pay out individuals or groups for task fulfillment
+
+contract StandardBounty {
+
+  /*
+   * Storage
+   */
+
+  address public masterCopy;
```

Comment:
This took me a second. but I get it. It's pretty cool - so this code is what updates the masterCopy within each deployed proxy contract which ultimately points to the deployed master contract. very cool.